Sofa/Component/LinearSolver/Direct/src/sofa/component/linearsolver/direct/SparseLDLSolver.h

```diff
@@ -68,6 +68,8 @@ public :
         return new InvertData();
     }
 
+    Data<bool> d_parallelInverseProduct;
+
```

Comment:
```suggestion
```

I don't get why you added this. This Data is already in its base class in `MatrixLinearSolver`